packages/pn-personafisica-webapp/src/pages/Deleghe.page.tsx

```diff
@@ -1,6 +1,6 @@
-import { useEffect } from 'react';
+import * as React from 'react';
```

Comment:
Come mai l'import di tutto React?

Reply:
In seguito alle modifiche apportate al componente in oggetto (per adattarsi alla nuova logica di gestione degli errori gestisco il messaggio dell'errore tramite useState) ho dovuto fixare il relativo test mockando lo useState. Grazie a Carlos abbiamo scoperto che il mock dello useState funziona solamente a patto che all'interno del componente il codice sia espresso nel formato `React.useState('')`